asab/library/providers/zookeeper.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import kazoo.exceptions
+import urllib.parse
+import asab.zookeeper.container
+
+from .abc import LibraryProviderABC
+
+#
+
+L = logging.getLogger(__name__)
+
+#
+
+
+class ZooKeeperLibraryProvider(LibraryProviderABC):
+
+
+	def __init__(self, app, path, encoding="utf-8"):
+		super().__init__(app, path)
+		self.App = app
+		self.Encoding = encoding
+		self.Path = path
+		url_pieces = urllib.parse.urlparse(self.Path)
+		self.BasePath = url_pieces.path
+		if self.BasePath.endswith("/"):
+			self.BasePath = self.BasePath[:-1]
+
+		# Initialize ZooKeeper client
+		self.ZookeeperContainer = asab.zookeeper.container.ZooKeeperContainer(
+			self.App,
+			config_section_name='',
+			z_path=path
+		)
```

Comment:
Please rebase to https://github.com/TeskaLabs/asab/pull/213 and test that.
Here you need to use `ZooKeeperService.build_container(...)`, don't construct the container on your own.